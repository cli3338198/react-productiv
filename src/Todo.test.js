import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo.js"
import todos from "./_testData_.js"

describe("renders a single todo", function () {
  it("renders without crashing", function () {
    render(<Todo {...todos[0]} />);
  });

  it("matches snapshot", function(){
    const {container} = render(<Todo {...todos[0]} />);
    expect(container).toMatchSnapshot();
  });
});
