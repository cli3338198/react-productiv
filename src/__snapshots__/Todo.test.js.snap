// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a single todo matches snapshot 1`] = `
<div>
  <div
    class="Todo"
    id="1"
  >
    <div>
      <b>
        t1
      </b>
       
      <small>
        (priority: 
        2
        )
      </small>
    </div>
    <div>
      <small>
        d1
      </small>
    </div>
  </div>
</div>
`;
